synapse/rest/client/register.py

```diff
@@ -330,11 +330,11 @@ def __init__(self, hs: "HomeServer"):
                 # Artificially delay requests if rate > sleep_limit/window_size
                 sleep_limit=1,
                 # Amount of artificial delay to apply
-                sleep_msec=1000,
+                sleep_delay=1000,
                 # Error with 429 if more than reject_limit requests are queued
                 reject_limit=1,
                 # Allow 1 request at a time
-                concurrent_requests=1,
+                concurrent=1,
```

Comment:
This is a functional change too? I think its fine though

Reply:
Sorry, forgot to point it out